src/librustc/infer/error_reporting/different_lifetimes.rs

```diff
@@ -204,6 +204,13 @@ impl<'a, 'gcx, 'tcx> Visitor<'gcx> for FindNestedTypeVisitor<'a, 'gcx, 'tcx> {
 
     fn visit_ty(&mut self, arg: &'gcx hir::Ty) {
         match arg.node {
+            hir::TyBareFn(_) => {
+                self.depth += 1;
+                intravisit::walk_ty(self, arg);
+                self.depth -= 1;
+                return;
+            }
+            
```

Comment:
Trailing whitespace

```
[00:04:19] tidy error: /checkout/src/librustc/infer/error_reporting/different_lifetimes.rs:213: trailing whitespace
```

BTW you could run the tidy test locally using 

```sh
./x.py test src/tools/tidy
```

Reply:
You also need to handle `TyTraitObject` (aka the `Fn` in `Box<Fn<(&u8, &u8)>>`).

Reply:
yeah at it :)

Reply:
@arielb1 need an example to test